gems/pending/appliance_console/internal_database_configuration.rb

```diff
@@ -171,7 +172,9 @@ def block_until_postgres_accepts_connections
     end
 
     def create_postgres_root_user
-      run_as_postgres("psql -c 'CREATE ROLE #{username} WITH LOGIN CREATEDB SUPERUSER PASSWORD '\\'#{password}\\';")
+      conn = PG.connect(:user => "postgres", :dbname => "postgres")
+      esc_pass = conn.escape_string(password)
+      conn.exec("CREATE ROLE #{username} WITH LOGIN CREATEDB SUPERUSER PASSWORD '#{esc_pass}'")
```

Comment:
You could further improve and simplify this fix by calling `exec_params` instead of `exec`.

See <http://deveiate.org/code/pg/PG/Connection.html#method-i-exec_params>